src/utils/queryString/parseQueryString.ts

```diff
@@ -9,8 +9,5 @@ export const parseQueryString = (search: string): ParseQueryString =>
 export const getString = (param: string | string[] = ""): string =>
   Array.isArray(param) ? param[0] : param;
 
-export const getArray = (param: string | string[] = []): string[] =>
```

Comment:
`toArray` does the same thing and is more generic instead of focusing on just queryStrings